app/services/create_member_for_api_members_controller.rb

```diff
@@ -27,6 +27,12 @@ def create
 
   private
 
+  def member_params
+    return @member_params if @member_params.present?
+    allowed = Member.new.attributes.keys.map(&:to_sym) + [:name]
+    @member_params = @params.select{ |k, v| allowed.include? k }
```

Comment:
You're also permitting params in the controller - does this need to be here, as well?